<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-color: qconicalgradient(cx:1, cy:1, angle:0, stop:0 rgba(108, 99, 99, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>471</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CAPTURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearDataButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR ALL
DATA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="eggCB">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Balut</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Penoy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Abnoy</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="rgbButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binaryButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BINARY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EQUALIZED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold
Pixels:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshPixelsTB">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="modeCB">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10 days mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14 days mode</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>eggCB</tabstop>
  <tabstop>captureButton</tabstop>
  <tabstop>clearDataButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
